user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -279,7 +280,9 @@ DataMiner supports Okta as identity provider as from version 10.1.11. Use Okta's
 > [!IMPORTANT]
 >
 > - Prior to DataMiner 10.3.0/10.3.2, it may not be possible to log in using Okta because of a software issue. We strongly recommend that you upgrade to DataMiner 10.3.0 or 10.3.2 to use this feature.
-> - DataMiner only supports the use of automatic user creation with Okta.
+> - When Okta is used, automatic user creation must be enabled. It is currently not possible to import users and groups from Okta.
+> - Alternatively you can add local or domain users, see "User directories" and "Local users" under [User Management] (xref:User_management).
+>   After that you can point Okta to authenticate these users by following the below guide but ommitting the AutomaticUserCreation tag from Dataminer.xml
```

Comment:
Can these three lines be combined? I'd suggest to combine them as follows:
`> - When Okta is used, automatic user creation must be enabled. It is currently not possible to import users and groups from Okta. Alternatively, you can add local or domain users. See [User directories](xref:User_management#user-directories) and [Local users](xref:User_management#local-users). After that, you can have Okta authenticate these users by following the guide below, except that you omit the *AutomaticUserCreation* tag in *DataMiner.xml*.`

Reply:
done